principles-of-providing-excellent-support/frictionless-support.md

```diff
@@ -0,0 +1,85 @@
+
+Even if the person contacting us is on a tight deadline and has tremendous urgency, it does not fit the criteria for the urgent tag unless the site was previously accepting donations and now is no longer. 
+
+By this definition, a site that is not live yet or has never taken donations before can't have an urgent-tagged issue. 
+
+### Responding to Urgent tagged tickets
+
+When a ticket is marked as urgent (according to the above definition) and left in the unassigned box, all technicians on the unassigned box are to prioritize it. 
+
+That doesn't necessarily mean that they should be the one to answer, but that they should alert the technician who tagged it to answer when there is a response from the customer. 
+
+If the original technician is out for the day at the time of the customer response, the remaining technician(s) should take over the ticket, and respond. As long as the issue is still urgent, we leave the ticket in the unassigned box tagged as urgent. 
+
+## Escalating Tickets To Senior Support Technicians
+
+Another aspect of frictionless support is escalating tickets to Senior Support Technicians. The goal is to only respond to the customer with actionable, helpful, and constructive responses. 
+
+To say "I am looping in a Senior support technician" to the customer adds friction, and should only be used as a [Queue Health Habit](../daily-routine/queue-health-habits.md) to get a response to the customer before logging off for the day. It would be far better to have a conversation with the Senior Support Technician and then respond with a fix for the issue. 
+
+### Mechanics of Escalation
+
+The point of escalating a ticket is to signal to a Senior Support Technician "I don't know what else to do to resolve this issue." For mechanics of _how_ to do certain tasks, [Slack calls](../communicating-as-a-team/using-slack.md) are a better choice. 
+
+If a Support Technician finds themselves stuck on what else to do, we escalate the ticket via a note in Help Scout. The singular focus of that note is to bring the Senior Support Technician up to speed on the ticket without having to read through the entire thing. We use the canned response in Help Scout to answer most thoroughly. 
+
+Once the ticket has been escalated, **we leave it assigned to the original technician** unless otherwise instructed. This allows for the technician to monitor whether the Senior Support Technician has gotten to it, and if they haven't, to either remind them or notify the customer before logging off for the day. 
+
+### Angry Customers
+The other reason for escalating a ticket is if the customer is demonstrably angry or belligerent. Often an angry customer (in addition to simply needing to be handled per our [tone guide](tone-guide.md)'s principle of "results oriented") simply needs to hear from a different technician, and they respond well to titles like "Senior," "Director," or "Head." 
```

Comment:
This is a public doc, so I think this paragraph needs to be massaged a bit. The part about "they respond well to title's" is a bit too belittling. I think this should focus on "the Head of Support (or Senior Support Technician) should be notified about any and all vocally dissatisfied customers as soon as possible." That's really the main action we want to take in these circumstances.